src/posix/platform/radio_spinel.cpp

```diff
@@ -752,6 +785,17 @@ void RadioSpinel::Process(const fd_set &aReadFdSet, const fd_set &aWriteFdSet)
             RadioTransmit();
         }
     }
+
+    // handle radio tx timeout
+    {
+        uint64_t now = otSysGetTime();
+
+        if (now > mTxRadioEndUs && mState == kStateTransmitting)
+        {
+            otLogCritPlat("radio tx timeout, exit");
+            exit(OT_EXIT_FAILURE);
+        }
+    }
```

Comment:
How about this to avoid extar `{ }`:
```
    // handle radio tx timeout
    if ((mState == kStateTransmitting) && (otSysGetTime() > mTxRadioEndUs))
    {
        otLogCritPlat("radio tx timeout, exit");
        exit(OT_EXIT_FAILURE);
    }
````

Reply:
Yes, maybe we could move this block to line 799 and use `else if`